src/main/scala/sbtorgpolicies/settings/AllSettings.scala

```diff
@@ -136,6 +137,12 @@ trait AllSettings
     )
   }
 
+  lazy val scalaMetaSettings = Seq(
```

Comment:
A small note on the context: I first found the reference for these settings in this [tutorial](http://scalameta.org/tutorial/#Macroannotations).